app/models/katello/repository.rb

```diff
@@ -84,6 +84,8 @@ class Repository < Katello::Model
       :allow_blank => false,
       :message => ->(_, _) { _("must be one of the following: %s") % Katello::RepositoryTypeManager.repository_types.keys.join(', ') }
     }
+    validates :download_policy, inclusion: { in: ::Runcible::Models::YumImporter::DOWNLOAD_POLICIES }, if: :yum?
+    validate :ensure_no_download_policy, if: ->o { !o.yum? }
```

Comment:
Yea, I was trying to be consistent with [the existing code](https://git.io/vgfrX). I suppose I can update those lines too though.